README.md

```diff
@@ -72,3 +72,27 @@ REACT_APP_TORNJAK_MANAGER=true npm start
 ```
 
 In this view, there is an additional navigation bar tab titled "Manage Servers" where you may register Tornjak agents.  
+
+### Running Tornjak With Auth
+Once you have Auth instance running: 
+Example - Keycloak instance running locally or on the cloud 
+
+To run Keycloak locally, run:
+
+```docker run -p 8080:8080 -e KEYCLOAK_ADMIN=”username” -e KEYCLOAK_ADMIN_PASSWORD=”password” quay.io/keycloak/keycloak:19.0.1 start-dev
+```
+
+this will start a listening port on 8080 for the keycloak instance. 
+
+To start the UI with Auth service, run:
+
+```
+REACT_APP_API_SERVER_URI=http://localhost:10000/
+REACT_APP_AUTH_SERVER_URI=http://localhost:8080/ npm start
+```
+To run frontend on a container as a seperate image with Auth:
+
+```
+AUTH_SERVER_URI=http://localhost:8080 APP_SERVER_URI=http://localhost:10000 CONTAINER_TAG=tsidentity/tornjak-spire-server:latest make container-frontend-push
+```
+Depending on the user used to signin, there will be two different views. And Admin User will have an Admin portal with admin privilages and a viewer user will have restricted access for the Tornjak UI only for viewing.  
```

Comment:
for local case in this readme, maybe add a sentence on running the container locally with a `docker run` command with the port mapping and where to access it?

Reply:
done